reviews/npm/mime/review-1.md

```diff
@@ -0,0 +1,51 @@
+---
+Publication-State: Draft
+Access: Public
+Reviewers:
+- Name: Dilan Bhalla
+  Associated-With-Project: false
+  Compensation-Source: none
+Domain: Security
+Methodology:
+- Static-Analysis
+- Web-Search
+Issues-Identified: Non-Severe
+Package-URLs:
+- pkg:npm/mime@1.3.6
```

Comment:
Would you mind updating the Package URLs to include the full list? According to [this](https://www.npmjs.com/advisories/535/versions), it should be:
- pkg:npm/mime@1.0.0
- pkg:npm/mime@1.1.0
- pkg:npm/mime@1.2.1
- pkg:npm/mime@1.2.10
- pkg:npm/mime@1.2.11
- pkg:npm/mime@1.2.2
- pkg:npm/mime@1.2.3
- pkg:npm/mime@1.2.4
- pkg:npm/mime@1.2.5
- pkg:npm/mime@1.2.6
- pkg:npm/mime@1.2.7
- pkg:npm/mime@1.2.8
- pkg:npm/mime@1.2.9
- pkg:npm/mime@1.3.0
- pkg:npm/mime@1.3.1
- pkg:npm/mime@1.3.2
- pkg:npm/mime@1.3.3
- pkg:npm/mime@1.3.4
- pkg:npm/mime@1.3.5
- pkg:npm/mime@1.3.6
- pkg:npm/mime@1.4.0
- pkg:npm/mime@2.0.1
- pkg:npm/mime@2.0.2

Reply:
For sure, just updated it.